Assets/HoloToolkit/Utilities/Scripts/ReadOnlyHashSet.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
ReadOnlyHashSet isn't necessary for input to work, but we found it useful while developing our app. I'm fine keeping it or removing it. Anyone feel strongly either way?